neard/bin/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+name = "neard-bin"
```

Comment:
Let's keep using `neard` as the name of the binary. Otherwise a bunch of docs and code that rely on this will break

Reply:
Understood. And yeah, that's the plan.
Although within the workspace, a lot equally depends on `neard` as the lib name. With the _first pass_, it's like this for now while we make sure we have a proper structure for where we want the lib to go.

The options available are:
- Have the lib named `nearcore`, for which we have two options:
  - Move the lib into the root crate.
  - Rename the workspace.
- Rename the lib. I mentioned something like `libnear` but @frol rightly pointed out that seems more appropriate for a C interop we might not want to block by taking that name.

WDYT?